.github/workflows/it-ex.yml

```diff
@@ -0,0 +1,38 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+on:
+  workflow_call:
+    inputs:
+      druid_it_image_name:
+        required: true
+        type: string
+jobs:
+  it:
+    strategy:
+      fail-fast: false
+      matrix:
+        #jdk: [8, 11, 17]
+        jdk: [8]
+        it: [HighAvailability, MultiStageQuery, Catalog, BatchIndex]
+        #indexer: [indexer, middleManager]
+        indexer: [middleManager]
+    uses: tejaswini-imply/druid/.github/workflows/reusable-revised-its.yml@update_uts_gha
+    with:
+      build_jdk: ${{ matrix.jdk }}
+      runtime_jdk: ${{ matrix.jdk }}
+      use_indexer: ${{ matrix.indexer }}
```

Comment:
Thanks for raising this, Currently there are 7 groupings of standard ITs (not including k8s tests)
-  integration-index-tests-middleManager
- integration-index-tests-indexer
- integration-query-tests-middleManager
- integration-query-tests-middleManager-mariaDB
- integration-shuffle-deep-store-tests
- integration-custom-coordinator-duties-tests
This is slightly arbitrary (e.g. security, high-availability are grouped under integration-query-tests-middleManager since they have same variable requirements as of that group), but this naming provide more or less decent info on where a new group has to be placed, if not a new group can be created with required config vars. But in the long run I believe `indexers.txt` approach could be followed without any need of grouping and this whole workflow script can be automated to just 3-4 lines.